main.css

```diff
@@ -1,33 +1,116 @@
-.card {
-    width: 300px;
+
+body {
+    margin: 0px;
+    box-sizing: border-box;
+    overflow-x: hidden;
+}
+
+footer {
+    background-color: rgba(71, 69, 66, 0.5);
+    padding: 30px;
+    margin: 50px 0 0 0;
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+}
+
+.footer__copyright {
```

Comment:
@Mmarvinn `copyright` is an element, not modifier

Reply:
fix